source/components/ansible-setup.rst

```diff
@@ -165,6 +165,12 @@ New databroker configuration
 Add a file to ``roles/databroker_config/files/`` and deploy the beamline
 playbook.
 
+.. note::
```

Comment:
It's worth considering changing `note` to `warning` and removing the shout-y "BE CAREFUL:"

Reply:
I have replaced this, although personally I think the shout-y "BE CAREFUL" is more likely to be noticed than a 'warning'.